Helper/Order.php

```diff
@@ -970,11 +974,6 @@ private function holdOnTotalsMismatch($order, $transaction)
                 ]
             ]);
         });
-
-        throw new LocalizedException(__(
```

Comment:
I think we still want to return error here so merchant is notified via failed hook? WDYT?

Reply:
I think we still want this as well. Merchants are constantly asking for more notifications on failed events during customer calls, so keeping it in is valuable I think.